main.tf

```diff
@@ -11,7 +11,7 @@ resource "helm_release" "argocd" {
   namespace        = try(var.argocd.namespace, "argocd")
   create_namespace = try(var.argocd.create_namespace, true)
   chart            = try(var.argocd.chart, "argo-cd")
-  version          = try(var.argocd.chart_version, "5.51.6")
+  version          = try(var.argocd.chart_version, "6.5.0")
```

Comment:
Jumping from 5.x to 6.x what are the breaking changes in the helm chart?
Anything we need to update on the examples?